app/models/upvs/message_template.rb

```diff
@@ -106,6 +106,12 @@ def build_message_from_template(message)
       filled_content.gsub!(template_item[:placeholder], message.metadata['data'][template_item[:name]].to_s)
     end
 
+    message.update(
+      html_visualization: message.metadata["data"]&.map do |k,v|
+        "#{k}: #{v}"
+      end&.join(', ')
+    )
+
```

Comment:
Tieto & vlaciky sa mi nikdy nepacili. Nepomohlo by sa tu len spytat, ci mame message.metadata["data"] a potom proste bez toho?

```suggestion
    message.update(
      html_visualization: message.metadata["data"]..map { _1.join(": ") }.join(", ")
    ) if message.metadata["data"]

```